src/verification/verification.controller.ts

```diff
@@ -0,0 +1,11 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { VerificationService } from '../verification/verification.service';
+
+@Controller('/verification')
+export class VerificationController {
+  constructor(private readonly verificationService: VerificationService) {}
+  @Get('/:token')
```

Comment:
- This should not be a Get request. This should be a PUT request. you can place token inside body
- This does not have to be in a separate controller. I would place it in the user controller,